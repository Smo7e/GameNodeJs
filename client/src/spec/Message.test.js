import React from 'react';
import { render } from '@testing-library/react';
import Message from '../component/Interface/component/Message/Message';

describe('Message Component', () => {
  it('renders correctly with message and name props', () => {
    const { asFragment } = render(<Message message="Hello world!" name="John Doe" />);
    expect(asFragment()).toMatchSnapshot();
  });
});

