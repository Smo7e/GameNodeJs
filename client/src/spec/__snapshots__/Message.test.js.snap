// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Component renders correctly with message and name props 1`] = `
<DocumentFragment>
  <div
    class="Message-Container"
  >
    <div
      class="message-name"
    >
      John Doe: 
    </div>
    <div
      class="message"
    >
      Â Hello world! 
    </div>
  </div>
</DocumentFragment>
`;
